charts/internal/cloud-controller-manager/templates/cloud-controller-manager.yaml

```
@@ -117,6 +117,12 @@ spec:
           mountPath: /var/lib/cloud-controller-manager-server
         - name: cloud-provider-config
           mountPath: /etc/kubernetes/cloudprovider
+      # Host certificates are mounted to accommodate OpenStack endpoints that might be served with a certificate
+      # signed by a CA that is not globally trusted.
+        - name: usr-share-ca-certificates
+          mountPath: /usr/share/ca-certificates
+        - name: etc-ssl
+          mountPath: /etc/ssl
```

Would it make sense to mount them as `readOnly` here, too?
```suggestion
        - name: usr-share-ca-certificates
          mountPath: /usr/share/ca-certificates
          readOnly: true
        - name: etc-ssl
          mountPath: /etc/ssl
          readOnly: true
```